### PR TITLE
Preparing release 14.6.0

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Review by matt-boris — APPROVED** [review]

File: kinto/plugins/admin/package.json [hunk 1]
```diff
@@ -1,21 +1,15 @@
...
-    "codemirror": "^5.63.3",
-    "kinto-admin": "1.30.2",
-    "lodash.isequalwith": "^4.4.0",
+    "kinto-admin": "1.31.0",
     "react": "^16.8.6",
-    "react-dom": "^16.8.6",
-    "react-router": "^5.2.1",
-    "redux": "^4.1.2",
-    "typescript": "^4.4.4"
+    "react-dom": "^16.8.6"
```
Comment (grahamalama) [inline kinto/plugins/admin/package.json:12]: I think we need to bump these react versions to `17.0.2` per https://github.com/Kinto/kinto-admin/pull/2055

Comment (leplatrem) [reply kinto/plugins/admin/package.json:12]: I think I need to take a course on node packaging :cry: 

```
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.0.0 || ^16.0.0" from react-breadcrumbs@2.1.7
npm ERR! node_modules/react-breadcrumbs
npm ERR!   react-breadcrumbs@"^2.1.6" from kinto-admin@1.31.0
npm ERR!   node_modules/kinto-admin
npm ERR!     kinto-admin@"1.31.0" from the root project
```

Comment (grahamalama) [reply kinto/plugins/admin/package.json:12]: It's toasty down here in dependency hell 🔥

**Comment by leplatrem** [comment]
I'm going to postpone the upgrade of kinto-admin for now.

**Pull request merged**